awx/ui/src/contexts/Session.js

```diff
@@ -110,10 +110,12 @@ function SessionProvider({ children }) {
     clearTimeout(sessionTimeoutId.current);
     clearInterval(sessionIntervalId.current);
 
+    const calcTimeOut = calcRemaining() - SESSION_WARNING_DURATION * 1000;
+
     isSessionExpired.current = false;
     sessionTimeoutId.current = setTimeout(
       handleSessionTimeout,
-      Math.min(calcRemaining() - SESSION_WARNING_DURATION * 1000, MAX_TIMEOUT)
```

Comment:
I tested this again. We were using a MAX_TIMEOUT for Math.min, and then the other value was always being returned. If I missed something let me know.

Reply:
Make sure to consider multiple tabs, etc. Is there a specific reason we're removing this?